examples/EvaluationDataToApplicationInsights/appsettings.json

```diff
@@ -0,0 +1,53 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ApplicationInsights": {
+    "ConnectionString": "<connection-string>"
```

Comment:
Can you put a readme of what a user needs to do to get this sample working?

Reply:
Yeah, good idea

Reply:
Added in https://github.com/microsoft/FeatureManagement-Dotnet/pull/284/commits/9a4370bbb83f522d57b789a1d0df549d66c880bc?short_path=dc904d4#diff-dc904d4dce5d3420435595d0fb7154a78d5cd82d78e02cc0e1d262266869af20